debian/sid/debian/patches/funciton.patch

```diff
@@ -0,0 +1,18 @@
+Description: Fix funciton typo
+Author: Ben Albrecht <balbrecht@cray.com>
+Origin: backport, https://github.com/chapel-lang/chapel/pull/5964
```

Comment:
We may want to include the [forwarded](http://dep.debian.net/deps/dep3/) field to make it clear that this patch has been made upstream.